config-laptop.plist/kaby-lake.md

```diff
@@ -123,66 +123,52 @@ This section is set up via WhateverGreen's [Framebuffer Patching Guide](https://
+* **Port Count**
+  * The number of displays supported
 
 Generally follow these steps when setting up your iGPU properties. Follow the configuration notes below the table if they say anything different:
 
 1. When initially setting up your config.plist, only set AAPL,ig-platform-id - this is normally enough
-2. If you boot and you get no graphics acceleration (7MB VRAM and solid background for dock), then you likely need to set device-id as well
-
-Note that highlighted entries with a star(*) are the recommended entries to use:
-
-| iGPU | device-id | AAPL,ig-platform-id | Port Count | Total Stolen Memory | Connectors |
-| :--- | :--- | :--- | :--- | :--- | :--- |
-| **Intel HD Graphics 620** * | 59160000 | 00001659 | 3 | 35MB |  LVDSx1 DPx2 |
-| Intel HD Graphics 620 | 59160000 | 09001659 | 3 | 39MB |  LVDSx1 DPx2 |
-| **Intel HD Graphics 630** * | 591B0000 | 00001B59 | 3 | 39MB |  LVDSx1 DPx2 |
-| Intel HD Graphics 630 | 591B0000 | 06001B59 | 1 | 39MB |  LVDSx1 |
-| Intel HD Graphics 615 | 591E0000 | 00001E59 | 3 | 35MB |  LVDSx1 DPx2 |
-| Intel HD Graphics 615 | 591E0000 | 01001E59 | 3 | 39MB |  LVDSx1 DPx2 |
-| Intel Iris Plus Graphics 650 | 59270000 | 04002759 | 3 | 58MB |  LVDSx1 DPx2 |
-| Intel Iris Plus Graphics 650 | 59270000 | 09002759 | 3 | 39MB |  LVDSx1 DPx2 |
-| **Intel UHD Graphics 617** * | 87C00000 | 0000C087 | 3 | 35MB |  LVDSx1 DPx2 |
-| Intel UHD Graphics 617 | 87C00000 | 0500C087 | 3 | 58MB |  LVDSx1 DPx2 |
+2. If you boot and you get no graphics acceleration (7MB VRAM and solid background for dock), then you likely need to try different AAPL,ig-platform-id values add stolenmem patches or even add a device-id
+
+| AAPL,ig-platform-id | Port Count | Comment |
+| ------------------- | ---------- | ------- |
+| **00001B59** | 3 | Recommended for HD615, HD630, HD640 and HD650 |
+| **00001659** | 3 | Alternative value to 00001B59 if you have acceleration issues |
+| **0000C087** | 3 | Recommended for Amber lake's UHD 617 and Kaby lake R's UHD620 |
 
 #### Configuration Notes
 
-* **Note:** `UHD630` ***IS NOT*** KabyLake, it's CoffeeLake (check next section).
-* For `HD615`, `HD630`, `HD640` and `HD650` it is not needed to use a `device-id`. However, due to many issues with different setups it is recommended to use:
-  * `device-id`=`591b0000` or `59160000`
-  * `AAPL,ig-platform-id`=`00001659` or `00001b59` (you can try whichever works the best, some even try to cross the device-id and the ig-platform-id)
-* For `UHD620` users we recommend the following values:
-  * `device-id`=`59160000`
-  * `AAPL,ig-platform-id`=`0000C087`
+* For `UHD620` users(Kaby lake R), you'll need a device-id spoof:
+
+| Key | Type | Value |
+| :--- | :--- | :--- |
+| device-id | Data | 16590000 |
+
 * For all HD6\*\* (`UHD` users are not concerned), there are some small issues with output where plugging anything would cause a lock up (kernel panic), here are some patches to mitigate that (credit Rehabman):
```

Comment:
```suggestion
* For all HD6\*\* (`UHD` users are not concerned), there are some small issues with output where plugging anything would cause a lock up (kernel panic); here are some patches to mitigate that (credit Rehabman):
```